File: test/fixtures/csharp/get_charles_syntax.cs
using System.Net.Http;

HttpClientHandler handler = new HttpClientHandler();
handler.AutomaticDecompression = DecompressionMethods.All;

HttpClient client = new HttpClient(handler);

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/?format=json&");

request.Headers.Add("Host", "api.ipify.org");
request.Headers.Add("Accept", "*/*");
request.Headers.Add("User-Agent", "GiftTalk/2.7.2 (iPhone; iOS 9.0.2; Scale/3.00)");
request.Headers.Add("Accept-Language", "en-CN;q=1, zh-Hans-CN;q=0.9");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/php/get_charles_syntax.php
<?php
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://localhost:28139/?format=json&');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'GET');
curl_setopt($ch, CURLOPT_HTTPHEADER, [
    'Host: api.ipify.org',
    'Accept: */*',
    'User-Agent: GiftTalk/2.7.2 (iPhone; iOS 9.0.2; Scale/3.00)',
    'Accept-Language: en-CN;q=1, zh-Hans-CN;q=0.9',
    'Accept-Encoding: gzip',
]);

$response = curl_exec($ch);

curl_close($ch);
